app/views/users/show.html.erb

```diff
@@ -233,6 +237,30 @@
 
 <div class="richtext text-break clearfix"><%= @user.description.to_html %></div>
 
+<% if @heatmap_data.present? %>
+  <div class="row">
+    <div class="col overflow-auto">
+      <div id="cal-heatmap-wrapper" class="heatmap-wrapper">
+        <!-- Labels -->
+        <ul class="list-unstyled d-flex flex-column justify-content-between ch-domain-text heatmap-labels">
+          <li>&nbsp;</li>
+          <li><%= t("date.abbr_day_names")[1] %></li>
+          <li>&nbsp;</li>
+          <li><%= t("date.abbr_day_names")[3] %></li>
+          <li>&nbsp;</li>
+          <li><%= t("date.abbr_day_names")[5] %></li>
+          <li>&nbsp;</li>
+        </ul>
+        <!-- Heatmap -->
+        <div id="cal-heatmap" class="heatmap"
+          data-heatmap="<%= @heatmap_data.to_json %>"
+          data-site_color_scheme="<%= preferred_color_scheme(:site) %>">
```

Comment:
The color scheme is already present in `data-bs-theme` on `<html>` ("auto" if that attribute is missing).